metadoc-js/src/main/scala/monaco/Monaco.scala

```diff
@@ -80,7 +80,7 @@ class Promise[+V] protected () extends js.Object {
       init: js.Function3[TValueCallback[V], js.Function1[js.Any, Unit], ProgressCallback, Unit],
       oncancel: js.Any = ???
   ) = this()
-  def then[U](
+  def `then`[U](
```

Comment:
Why does this have to be backticked?

Reply:
deprecation warning about `then` becoming keyword :/